### PR TITLE
[PAN-2652] RefactorPrivacyTests

### PR DESCRIPTION
**Pull request by ekellstrand**
- Generate Account and Contract Hex instead of relying on literals
- Started conversion to a model that can generate keys & config, and support many nodes

### CODE REVIEW

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/privacy/PrivacyClusterAcceptanceTest.java [hunk 1]
```diff
@@ -13,505 +13,346 @@
...
-
-    privateTransactionVerifier
-        .validPrivateContractDeployed(CONTRACT_ADDRESS.toString())
-        .verify(node2, transactionHash);
-
-    transactionHash =
-        node2.execute(privateTransactions.createPrivateRawTransaction(storeValueFromNode2));
-
-    privateTransactionVerifier.validEventReturned("1000").verify(node1, transactionHash);
+  public PrivacyNode node(final int i) {
```
Comment (iikirilov) [inline acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/privacy/PrivacyClusterAcceptanceTest.java]: Helper methods can go in base class.

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/privacy/PrivacyNet.java [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+      stopOrionNodes();
+    } catch (RuntimeException e) {
+      LOG.error("Error stopping Orion nodes.  Logging and continuing.", e);
+    }
+  }
+
+  /** Verify that each Orion node has connected to every other Orion */
+  public void verifyAllOrionNetworkConnections() {
+    for (int i = 0; i < nodes.size(); i++) {
+      for (int j = i; j < nodes.size(); j++) {
```
Comment (iikirilov) [inline acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/privacy/PrivacyNet.java]: int j = i + 1

you do not need to test connectivity between node 0 and node 0...

**Comment by iikirilov** [comment]
replaced by #1483

**Comment by ekellstrand** [comment]
Closing as this is superseded by another PR that includes this and some other test enhancements.

**Pull request closed without merging**